<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastroCliente</class>
 <widget class="QDialog" name="cadastroCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela de Cadastro de Itens</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>10</y>
     <width>861</width>
     <height>521</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_senha">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_criarCliente">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_endereco">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_cpf">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_senha">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_endereco">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>EndereÃ§o</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_telefone">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_telefone">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSair">
    <property name="title">
     <string>Sair</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSair"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
